src/amo/components/ServerError/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import { oneLine } from 'common-tags';
+import { compose } from 'redux';
+import NestedStatus from 'react-nested-status';
+
+import Link from 'amo/components/Link';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML, visibleAddonType } from 'core/utils';
+import Card from 'ui/components/Card';
+
+import 'amo/components/ErrorPage/ErrorPage.scss';
+
+
+export class ServerErrorBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    const fileAnIssueText = i18n.gettext(oneLine`
```

Comment:
The text in each message is a bit tailored to the error, so it seems like more work to create a component that is essentially just filling in the blanks of `i18n.sprintf`... which admittedly isn't much of a lovely API, but adding a shared component for this seems overkill (it will be a lot more code/tests for something easy to do):

```js
const fileAnIssueText = i18n.sprintf(i18n.gettext(oneLine`
      If you are signed in and think this message is an error, please
      <a href="%(url)s">file an issue</a>. Tell us where you came from
      and what you were trying to access, and we'll fix the issue.`),
      { url: 'https://github.com/mozilla/addons-frontend/issues/new/' });
```